apps/frontend/src/views/Results.vue

```diff
@@ -355,6 +395,12 @@ export default class Results extends mixins(RouteMixin, ServerMixin) {
     this.search_term = '';
   }
 
+  mounted() {
+    if(this.serverMode) {
+      EvaluationModule.getAllEvaluations()
+    }
+  }
```

Comment:
I'm still a bit iffy on this change, I'd rather at least investigate if we can load only the evaluations we need instead of all of them.